### PR TITLE
321634 autocleanup in playground

### PR DESCRIPTION
**Pull request by magnus-longva-bouvet**

### CODE REVIEW

File: radix-cluster-cleanup/main.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"github.com/equinor/radix-cluster-cleanup/cmd"
+
 	// Force loading of needed authentication library
 	_ "k8s.io/client-go/plugin/pkg/client/auth/azure"
```
Comment (nilsgstrabo) [inline radix-cluster-cleanup/main.go:7]: You can remove k8s.io/client-go/plugin/pkg/client/auth/azure

It is deprecated. Ref https://github.com/kubernetes/client-go/blob/master/plugin/pkg/client/auth/azure/azure_stub.go

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
+FROM golang:1.18.5-alpine3.16 as builder
```
Comment (nilsgstrabo) [inline Dockerfile:1]: Use alpine 3.17

File: charts/radix-cluster-cleanup/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: 1.1.3
```
Comment (nilsgstrabo) [inline charts/radix-cluster-cleanup/Chart.yaml:2]: Should we start at version 1.0.0?

File: charts/radix-cluster-cleanup/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+description: A Helm chart for Kubernetes
+name: radix-cluster-cleanup
+version: 1.1.3
```
Comment (nilsgstrabo) [inline charts/radix-cluster-cleanup/Chart.yaml:5]: Same here. Start at version 1.0.0?

File: charts/radix-cluster-cleanup/templates/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "radix-cluster-cleanup.fullname" . }}
```
Comment (nilsgstrabo) [inline charts/radix-cluster-cleanup/templates/deployment.yaml:4]: missing namespace

File: .github/workflows/pull-request-check.yml [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+jobs:
+  build:
+    name: pull-request-check
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build docker image
+      env:
+        REF: ${{ github. sha }}
+      run: docker build -t radix-acr-cleanup:${REF##*/} .
```
Comment (nilsgstrabo) [inline .github/workflows/pull-request-check.yml:15]:
```suggestion
      run: docker build -t radix-cluster-cleanup:${REF##*/} .
```
Fix image name

**Review by nilsgstrabo — CHANGES_REQUESTED** [review]
Added a few suggestions

File: charts/radix-cluster-cleanup/Chart.yaml (hunk 3 above)
Comment (magnus-longva-bouvet) [reply charts/radix-cluster-cleanup/Chart.yaml:2]: yes, just had to increment versions during testing in test cluster

File: charts/radix-cluster-cleanup/templates/deployment.yaml [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+  namespace: radix-cluster-cleanup
```
Comment (nilsgstrabo) [inline charts/radix-cluster-cleanup/templates/deployment.yaml:5]:
```suggestion
  namespace: {{ .Release.Namespace }}
```
Namespace should use the namespace for the helm release, same as ServiceAccount

**Review by nilsgstrabo — CHANGES_REQUESTED** [review]
Fix hardcoded namespace

**Review by nilsgstrabo — APPROVED** [review]

**Pull request merged**